### PR TITLE
WIIP: Lyft case study v1 draft

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

**Comment by dadlerj** [comment]
Not ready for review, I assume?

**Comment by ryan-blunden** [comment]
> Not ready for review, I assume?

It's ready for Lyft to do their final review, but any other feedback related to code, punctuation, grammar etc. is welcome.

File: website/src/pages/case-studies/lyft-monolith-to-microservices.tsx [hunk 1]
```diff
@@ -58,30 +49,75 @@ hundreds of repositories, so a high level of accuracy is needed."
...
+
+                    <InContentBlockquote
+                        quote="Sourcegraph code search helped ensure production stability throughout the monolith to microservices decomposition by verifying applications were no longer pointing at the monolith version of a service."
+                        author="Justin Phillips, Software Engineer"
+                    />
+
+                    {/*This case study demonstrates how Lyft used Sourcegraph code search to aid in the decomposition of their 10 year old PHP monolith to microservices.*/}
+
+                    <p>
+                        This wasn't simply slicing up the monolith by domain into separate services—it was also a
```
Comment (vanesa) [inline website/src/pages/case-studies/lyft-monolith-to-microservices.tsx]: Add blank on either side of the dash between `services` and `it`.

Comment (ryan-blunden) [reply website/src/pages/case-studies/lyft-monolith-to-microservices.tsx]: It's an em dash so typically, there is no space either side.

**Pull request closed without merging**